area/area_blackbox_test.go

```diff
@@ -104,37 +82,18 @@ func (s *TestAreaRepository) TestCreateArea() {
 func (s *TestAreaRepository) TestExistsArea() {
 	t := s.T()
 	resource.Require(t, resource.Database)
+	repo := area.NewAreaRepository(s.DB)
 
 	t.Run("area exists", func(t *testing.T) {
 		// given
-		repo := area.NewAreaRepository(s.DB)
-		name := "TestCreateArea"
-		newSpace := space.Space{
-			Name:    uuid.NewV4().String(),
-			OwnerId: s.testIdentity.ID,
-		}
-		repoSpace := space.NewRepository(s.DB)
-		space, err := repoSpace.Create(context.Background(), &newSpace)
-		require.Nil(t, err)
-		a := area.Area{
-			Name:    name,
-			SpaceID: space.ID,
-		}
+		fxt := tf.NewTestFixture(s.T(), s.DB, tf.Areas(1))
 		// when
-		err = repo.Create(context.Background(), &a)
+		err := repo.CheckExists(context.Background(), fxt.Areas[0].ID.String())
```

Comment:
why do you want to keep that line ? Areas have been created, so I guess there's no need to verify it here

Reply:
It is a proof, that the function `CheckExists` actually works. So yes, I want to keep it.

Reply:
ok, fair enough ;)